source/patterns/@aws-solutions-constructs/core/lib/glue-database-defaults.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ *  Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+import { CfnDatabase } from "@aws-cdk/aws-glue";
+import { Aws, Construct } from "@aws-cdk/core";
+
+export function DefaultGlueDatabase(scope: Construct): CfnDatabase {
```

Comment:
Since this returns an instance of CDK object, it should be moved to `core/lib/glue-database-helper.ts`.. In general, `***-defults.ts` returns just the construct props with defaults, whereas `***-helper.ts`returns CDK objects